### ISSUE TITLE
start_data_store and start_fmke scripts need improvements

### ISSUE THREAD

**Issue by goncalotomas** [goncalotomas/FMKe#114]
Users trying out FMKe might want to try out multiple deployment scenarios, and it should be easier for them to use FMKe than with the current state of the included scripts.

Instead of keeping a `start_data_store.sh` and `start_fmke.sh` along with corresponding stop scripts, perhaps it would be easier to unify both into a `start_service.sh` or `start.sh`.

The parameter passed into the script would determine which of the behaviours is desired (`./start_service.sh fmke` would fire up fmke, but `./start_service.sh antidote` would only start antidote)

Other options include supporting opening a shell with a specific back-end (e.g. `./start_service.sh fmke antidote` would spin up FMKe with an antidote backend).

The purpose of this is to make it easier for users to try out FMKe, but also to add some robustness to these scripts.

If, to make things consistent we were to reimplement the scripts in pure Erlang, this could be done with use of the `os:cmd/1` function, although not so pretty.

Summing up, some ideas to improve the scripts are:

- Unify `start_data_store` and `start_fmke` into a single script
- Use more flexible options that allow for easy setup of FMKe and back end instances
- Use nodenames as parameters in order to support multiple instances per machine
- Ask user if database should be taken down when shutting down FMKe (application server)
- Ask if database is to be launched if user is trying to boot FMKe without any running database

These improvements are analogous for the scripts used to stop both types of services.
